openassessment/xblock/openassessmentblock.py

```diff
@@ -119,9 +119,21 @@ class OpenAssessmentBlock(
     )
 
     allow_file_upload = Boolean(
-        default=False,
+        default=None,
```

Comment:
@wmono do you remember why you changed this? I can't think of any reason we need this change as default value won't be used in all existing or new blocks anyway.